rust_src/src/character.rs

```diff
@@ -101,4 +101,20 @@ pub fn char_resolve_modifiers(character: LispObject) -> EmacsInt {
     char_resolve_modifier_mask(character.into())
 }
 
+/// Concatenate all the argument characters and make the result a string.
+/// usage: (string &rest CHARACTERS)
+#[lisp_fn]
+pub fn string(args: &[LispObject]) -> LispObject {
+    let n = args.len();
+    let mut buffer = vec![0_u8; n * MAX_MULTIBYTE_LENGTH];
```

Comment:
```suggestion
    let mut buffer = vec![0_u8; args.len() * MAX_MULTIBYTE_LENGTH];
```
did this because I see nowhere else `n` is used

Reply:
`n` is used in `make_string_from_bytes`.